tests/global_resources_test.py

```diff
@@ -0,0 +1,45 @@
+"""Test global resources system
+
+:copyright: Copyright (c) 2023 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+_PUBLIC_PORT = 12666
+
+
+def setup_module(_):
+    import os
+
+    os.environ.update(
+        SIREPO_FEATURE_CONFIG_ENABLE_GLOBAL_RESOURCES="1",
+        SIREPO_GLOBAL_RESOURCES_PUBLIC_PORTS_MAX=f"{_PUBLIC_PORT + 1}",
+        SIREPO_GLOBAL_RESOURCES_PUBLIC_PORTS_MIN=f"{_PUBLIC_PORT}",
+    )
+
+
+def test_global_resources(fc):
+    from pykern.pkunit import pkeq, pkok
+
+    d = fc.sr_sim_data()
+    a = _post(fc, "globalResources", d)
```

Comment:
I think we should not send sim_data. Instead we should request global resources for an existing stype and sid. The data should be read from sim_db if need be.